corehq/apps/app_manager/app_translations/generators.py

```diff
@@ -208,20 +211,29 @@ def occurrence(_row):
 
                 def occurrence(_row):
                     return _row[label_index]
+        is_module = type_and_id and type_and_id.type == "Module"
         for i, row in enumerate(rows):
+            index = i + 1
```

Comment:
Nitpick, but I'd inline the `i + 1` below instead of making a separate variable, because it's confusing to have both `i` and `index`. Or else rename `index` something more semantic like `form_or_module_index`.